Dockerfile

```diff
@@ -1,44 +1,27 @@
 ARG BASE_IMAGE=python:3.8.5-buster
```

Comment:
Would be good for Aditya to review the Dockerfile, yml, serge_ts_connector, and any important serge config changes, if he hasn't already.

Reply:
reviewed all good 👍